### DISCUSSION TITLE
Query: How to Hide/ Remove Tooltip

### DISCUSSION THREAD

**Discussion by CityBelfast** [geoman-io/leaflet-geoman#849] [Questions & Help]
Hi, is there any option where I can hide/ remove this below tooltip while drawing geometry? In fact, this tooltip hiding the actual line of drawing geometry.

![image](https://user-images.githubusercontent.com/76213912/106788061-6437f780-6648-11eb-9ce8-9e6a3d567365.png)

**Comment by Falke-Design** [comment]
The Tooltip can be hidden with the option `tooltips: false`

```
map.pm.setGlobalOptions({tooltips: false})
```
